DATA/production/qc-postproc-async/zdc.json

```diff
@@ -0,0 +1,226 @@
+          {
+            "name": "details",
+            "title": "ZDC DETAILS",
+            "path": "ZDC/QO",
+            "ignoreQualitiesDetails": [],
+            "inputObjects": [
+              {
+                "name": "QcZDCRecCheck",
+                "title": "Rec check"
+              }
+            ]
+          }
+        ],
+        "initTrigger": [
+          "userorcontrol"
+        ],
+        "updateTrigger": [
+          "newobject:qcdb:ZDC/QO/ZDCQuality/ZDCQuality"
+        ],
+        "stopTrigger": [
+          "userorcontrol"
+        ]
+      }
+    },
+    "checks": {
+      "RecCheck": {
+        "active": "true",
+        "className": "o2::quality_control_modules::zdc::ZDCRecDataCheck",
+        "moduleName": "QcZDC",
+        "detectorName": "ZDC",
+        "policy": "OnAny",
+        "checkParameters": {
+          "ADC_ZNAC": "700;200;400",
+          "ADC_ZNA1": "260;100;200",
+          "ADC_ZNA2": "300;100;200",
+          "ADC_ZNA3": "170;100;200",
+          "ADC_ZNA4": "170;100;200",
+          "ADC_ZNAS": "700;200;400",
+          "ADC_ZPAC": "700;200;400",
+          "ADC_ZPA1": "60;50;100",
+          "ADC_ZPA2": "80;50;100",
+          "ADC_ZPA3": "245;50;100",
+          "ADC_ZPA4": "700;200;400",
+          "ADC_ZPAS": "850;200;400",
+          "ADC_ZEM1": "1230;200;400",
+          "ADC_ZEM2": "1275;200;400",
+          "ADC_ZNCC": "700;200;400",
+          "ADC_ZNC1": "130;100;200",
+          "ADC_ZNC2": "280;100;200",
+          "ADC_ZNC3": "130;100;200",
+          "ADC_ZNC4": "260;100;200",
+          "ADC_ZNCS": "600;200;400",
+          "ADC_ZPCC": "790;200;400",
+          "ADC_ZPC1": "770;200;400",
+          "ADC_ZPC2": "350;100;200",
+          "ADC_ZPC3": "97;50;100",
+          "ADC_ZPC4": "67;50;67",
+          "ADC_ZPCS": "1044;200;400",
+          "ADC_POS_MSG_X": "0.15",
+          "ADC_POS_MSG_Y": "0.92",
+          "TDC_ZNAC": "0.0;2.0;4.0",
+          "TDC_ZNAS": "0.0;2.0;4.0",
+          "TDC_ZPAC": "0.0;2.0;4.0",
+          "TDC_ZPAS": "0.0;2.0;4.0",
+          "TDC_ZEM1": "0.0;2.0;4.0",
+          "TDC_ZEM2": "0.0;2.0;4.0",
+          "TDC_ZNCC": "0.0;2.0;4.0",
+          "TDC_ZNCS": "0.0;2.0;4.0",
+          "TDC_ZPCC": "0.0;2.0;4.0",
+          "TDC_ZPCS": "0.0;2.0;4.0",
+          "TDC_POS_MSG_X": "0.01",
+          "TDC_POS_MSG_Y": "0.92"
+        },
+        "dataSource": [
+          {
+            "type": "PostProcessing",
+            "name": "RecPP",
+            "MOs": [
+              "h_summary_ADC", "h_summary_TDC"
+            ]
+          }
+        ]
+      }
+    },
+    "aggregators": {
+      "ZDCQuality": {
+        "active": "true",
+        "className": "o2::quality_control_modules::common::WorstOfAllAggregator",
+        "moduleName": "QcCommon",
+        "policy": "OnAll",
+        "detectorName": "ZDC",
+        "dataSource": [
+          {
+            "type": "Check",
+            "name": "RecCheck"
+          }
+        ]
+      }
+    }
+  }
```

Comment:
@cpuggion84, there is one `}` missing. Please add it and reformat this json file.

Reply:
done